backend/db/migrate/20250807161839_add_investment_amount_to_dividends.rb

```
@@ -0,0 +1,5 @@
+class AddInvestmentAmountToDividends < ActiveRecord::Migration[8.0]
```

RuboCop (1.76.1)

[convention] Missing frozen string literal comment.

(Style/FrozenStringLiteralComment)